### PR TITLE
Provider 1.3.0 Release

### PR DESCRIPTION
**Pull request by mrm9084**
# Description

Updated Readme, version, and Changelog

### CODE REVIEW

File: sdk/appconfiguration/azure-appconfiguration-provider/README.md [hunk 1]
```diff
@@ -168,6 +168,19 @@ key_vault_options = AzureAppConfigurationKeyVaultOptions(
 config = load(endpoint=endpoint, credential=DefaultAzureCredential(), key_vault_options=key_vault_options)
 ```
 
+## Geo Replication
+
+The Azure App Configuration Provider library will automatically discover the provided configuration store's replicas at startup. If the store has any issue, the provider will automatically switch a replica if available. The provider will continue to use the replica until the store is available again.
```
Comment (avanigupta) [inline sdk/appconfiguration/azure-appconfiguration-provider/README.md:173]: We shouldnt be too specific about the implementation details. We can add a link to the main doc here: https://learn.microsoft.com/en-us/azure/azure-app-configuration/howto-geo-replication?tabs=dotnet#scale-and-failover-with-replicas

In the readme file, it should be good enough to say that the provider supports failover to replicas in case of failure.

**Review by avanigupta — APPROVED** [review]

File: sdk/appconfiguration/azure-appconfiguration-provider/README.md [hunk 2]
```diff
@@ -168,6 +168,21 @@ key_vault_options = AzureAppConfigurationKeyVaultOptions(
 config = load(endpoint=endpoint, credential=DefaultAzureCredential(), key_vault_options=key_vault_options)
 ```
 
+## Geo Replication
+
+The Azure App Configuration Provider library will automatically discover the provided configuration store's replicas and use the replicas if any issue arrises. From more information see [Geo-Replication](https://learn.microsoft.com/azure/azure-app-configuration/howto-geo-replication).
+
+) on MS Learn.
```
Comment (avanigupta) [inline sdk/appconfiguration/azure-appconfiguration-provider/README.md:175]: typo?

Comment (avanigupta) [reply sdk/appconfiguration/azure-appconfiguration-provider/README.md:175]: extra line and bracket

Comment (mrm9084) [reply sdk/appconfiguration/azure-appconfiguration-provider/README.md:175]: co-pilot. It auto added it. Thought I removed it when fixing the link.

**Review by avanigupta — APPROVED** [review]

File: sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py [hunk 3]
```diff
@@ -334,9 +307,33 @@ class AzureAppConfigurationProvider(Mapping[str, Union[str, JSON]]):  # pylint:
     keys. Enables resolution of Key Vault references in configuration settings.
     """
 
-    def __init__(self, endpoint, replica_client_manager, **kwargs) -> None:
+    def __init__(self, endpoint, **kwargs) -> None:
         self._origin_endpoint = endpoint
-        self._replica_client_manager = replica_client_manager
+
+        if "user_agent" in kwargs:
```
Comment (avanigupta) [inline sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py:313]: code changes seem big enough to do a separate PR explaining why we're changing this.

Comment (mrm9084) [reply sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py:313]: This is moving code from `_buildprovider` to inside the init.

**Review by avanigupta — APPROVED** [review]
